app/styles/components/events/view/export.scss

```diff
@@ -32,3 +32,11 @@
 .url {
   padding: 0 1em !important;
 }
+
+.segment pre {
```

Comment:
We already have x-overflow class, please use that.

Reply:
I am new here, can you further explain what is meant by that? 
Especially that I went through several scss files and "overflow" has been used before normally.
Thanks in advance.

Reply:
Hi @Elgayar , See here https://github.com/fossasia/open-event-frontend/blob/development/app/styles/partials/utils.scss. You can use the class `x-scrollable` class in the templates rather than specifying a new scss file.

Reply:
@kushthedude done, thanks for your help!